extensions/guacamole-auth-cas/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "LOGIN" : {
+        "FIELD_HEADER_TICKET" : ""
```

Comment:
I took a quick stab at simply not rendering fields or their headers if they have no associated content: mike-jumper/incubator-guacamole-client@6c5cdae

If that solves the case you're working around here, I'll open a pull request for that commit.